examples/hello-world/README.md

```diff
@@ -0,0 +1,17 @@
+# Hello World
+
+## How to use
+
+Within this directory, run `yarn` to install dependencies.
+
+To execute the Node.js program, run:
+
+```sh
+npx ncc run index.js
```

Comment:
We could also put these commands in `package.json#scripts`:

### `package.json`

```json
{
  "scripts": {
    "execute": "ncc run index.js",
    "build": "ncc build index.js -o dist"
  }
}
```

### `README.md`

```md
To execute the Node.js program, run `yarn execute`.
To build the Node.js program, run `yarn build`.
```

I chose [npx](https://www.npmjs.com/package/npx) as a way to execute `ncc` from within the `examples/hello-world` directory, but open to feedback on what would be helpful/simple.

Reply:
I like the idea of `scripts` better. `build` and `dev` IMO.